### PR TITLE
Tekdoc: Add support for v1beta1

### PR DESCRIPTION
**Pull request by chandanikumari**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Commit messages includes a project tag in the subject - e.g. [OCI], [hub], [results], [task-loops]

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
Thanks @chandanikumari

File: tekdoc/main.go [hunk 1]
```diff
@@ -19,79 +19,92 @@ package main
...
+// Use Go templates to print v1beta1.Task as Markdown
+
+func printTask(w io.Writer, task v1beta1.Task) error {
+	if task.Spec.Params != nil {
 
 		tmpl := template.Must(template.New("test").Parse(`# {{.Name}}
 ## Install the Task
-kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/{{.Name}}/{{.Name}}.yaml
-### Input:-
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/{{.Name}}/0.1/{{.Name}}.yaml
```
Comment (wlynch) [inline tekdoc/main.go:55]: nit: We shouldn't assume the version number is always 0.1. Fine for now, but we should look into deriving this from the yaml in a follow up PR. :D

**Review by wlynch — COMMENTED** [review]
/lgtm

**Pull request merged**